tic_tac_toe/game.py

```diff
@@ -24,7 +28,14 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    valid_positions = [
```

Comment:
nice visual representation of what positions are valid! much better than checking if len(position) == 2, etc...